src/components/calcite-color/calcite-color.e2e.ts

```diff
@@ -232,14 +232,15 @@ describe("calcite-color", () => {
     it("value as property", async () => {
       // initialize page with calcite-color to make it available in the evaluate callback below
       const page = await newE2EPage({
-        html: "<calcite-color></calcite-color>"
+        html: `<calcite-color id="temporary-picker"></calcite-color>`
       });
-      await page.setContent("");
```

Comment:
@jcfranco Is this cleanup acceptable? I was unable to get this test to pass when it removed the content like this.

Instead, the existing node is removed using evaluate.

Reply:
Is it failing on the CI or locally?

Reply:
It was failing both.

Reply:
It was throwing a `console.error` that is being caught.

Reply:
Weird, tests pass locally. I also checked the Travis builds and the previous broken builds seem unrelated to this change. We may be dealing w/ unstable tests.

Reply:
I'll revert so you can see it fail.

Reply:
Yeah maybe unstable tests because its now passing. 🤷